virttest/test_setup.py

```diff
@@ -66,14 +66,6 @@ class THPKhugepagedError(THPError):
     pass
 
 
-class HPNotSupportedError(Exception):
```

Comment:
I prefer to keep class HPNotSupportedError() although there is no other place to call it,
but it's still useful.

Reply:
If no other place call it, why keep it as a dead code ?  could you explain more about its use ?